packages/fxa-auth-server/lib/payments/configuration/base.ts

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import joi from '@hapi/joi';
+
+export type CapabilityConfig = {
+  [key: string]: string;
```

Comment:
This should be an array of strings per the schema in [FXA-3655](https://mozilla-hub.atlassian.net/browse/FXA-3655). Can you update this type and the corresponding Joi validator?

Reply:
Ah, good point, I actually wanted to change the schema defined there, but I made a mistake here as well (it should be an array of strings as you note!), I can update the issue to reflect the proposed change once its agreed upon.

@chenba The schema for this has a list of *either* individual strings or object(s?) with one or more client id's and an array of strings. I've changed this to an object so that all client-id's get a list of strings. The clientid of '*' is still assumed for all client-id's, though using some other marker seems fine to. I'd prefer special-casing the global capabilities, rather than specific client-id's, as providing the capability to all RPs is an edge case and would complicate parsing logic.

Reply:
Yes, it should be an array of strings.  👍